### PR TITLE
aligns encryption key injection with recent advances

### PR DESCRIPTION
**Pull request by xh3b4sd**

### CODE REVIEW

File: service/kvmconfig/v10/cloudconfig/master_template.go [hunk 1]
```diff
@@ -1,32 +1,54 @@
...
-
-	filesMeta := []k8scloudconfig.FileMetadata{
-		// Encryption key
-		{
-			AssetContent: encryptionConfig,
-			Path:         "/etc/kubernetes/encryption/k8s-encryption-config.yaml",
-			Owner:        FileOwner,
-			Permissions:  0600,
-		},
-	}
```
Comment (kopiczko) [inline service/kvmconfig/v10/cloudconfig/master_template.go]: LoL

**Review by kopiczko — APPROVED** [review]

File: service/kvmconfig/v10/cloudconfig/master_template.go (hunk 1 above)
Comment (xh3b4sd) [reply service/kvmconfig/v10/cloudconfig/master_template.go]: Yeah. I also giggled a bit.

File: service/kvmconfig/v10/version_bundle.go [hunk 2]
```diff
@@ -10,8 +10,8 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "component",
-				Description: "Add your change here.",
+				Component:   "cloudconfig",
+				Description: "Improved encryption key injection.",
```
Comment (kopiczko) [inline service/kvmconfig/v10/version_bundle.go]: Uncertain if we should present that to the customer. Security-wise it doesn't change anything (for kvm-operator).

Comment (xh3b4sd) [reply service/kvmconfig/v10/version_bundle.go]: IMO we should track everything in the changelogs. When there will be a requirement to filter changelogs for customers we can add this quite easily.

**Pull request merged**